django_perf_rec/settings.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding:utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from django.conf import settings
+
+
+class PerfRecSettings(object):
+
+    defaults = {
+        'MODE': 'once'
+    }
+
+    def get_setting(self, key):
+        try:
+            return settings.PERF_REC[key]
+        except:
```

Comment:
Never do `try / except` with bare `except`. This catches things like the program running out of memory, `KeyboardInterrupt`, etc. Use `AttributeError, IndexError` here.